### PR TITLE
Image property feature function implementation and tests

### PR DESCRIPTION
**Pull request by KangboLu**
I implemented my function and wrote 7 unit tests (1 for error handling testing) for my function output along with several pure color images for me to test my functions to get the color channel values. 

For the python version, I temporarily change it to ^3.7. You can change it back to ^3.8 if you want. It's just my conda is 3.7.10. For me to install 3.8, I will need to reinstall all the packages. If it works for 3.7, it will also work for 3.8.

> Before writing any code, revisit your function specifications from last week and revise them if needed.

I updated my function specification to reflect only the image properties. I think showing the histogram can be overkill for this project with a very limited amount of time for testing plots.

Since the unit test file is required for each function, I created a file for my function's unit test and separated my function from the picturepyfect.py file. The test will find the function as long as the function name is under the package folder: for our case, it's the picturepyfect folder. Separating the functions into separate files is actually cleaner and easier to maintain per function without the merge conflict from other peoples' functions.

> Write a unit test function for each specified function, which is named after the function being tested (e.g., if the function is named foo then the unit test function is named test_foo).

I tested my functions with pytest using `poetry run pytest` and all the tests passed. I can show you in our meeting.

image

### CODE REVIEW

File: picturepyfect/get_property.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    of RGB channels along with the file dimension and total pixels.
+    Parameters
+    ----------
+    image : numpy.ndarray
+        A n*n*3 numpy array to representing 3 or 4-channel RGB or RGBA image
+    Returns:
+    ---------
+    image_property: dictionary
+        a dictionary of image properties for dimension of width and height, total pixels,
+        and 3 channels' mean and median values separated by channel.
```
Comment (dusty736) [inline picturepyfect/get_property.py]: Maybe add a comment about how the mean and median values are in a single list?

**Review by dusty736 — COMMENTED** [review]
Just a quick comment on the docstring.  Looks great!

File: picturepyfect/get_property.py (hunk 1 above)
Comment (KangboLu) [reply picturepyfect/get_property.py]: Sure

**Comment by KangboLu** [comment]
@dusty736 I updated the docstring with a new commit. Please take a look. Thanks.

**Review by dusty736 — APPROVED** [review]
LGTM!

**Pull request merged**